manifests/params.pp

```diff
@@ -58,6 +64,33 @@
                                     '/usr/lib/syslinux/pxelinux.0']
       }
     }
+    /^(FreeBSD|DragonFly)$/: {
+      # if set to true, no repo will be added by this module, letting you to
+      # set it to some custom location.
+      $custom_repo         = true # as foreman packages are in standard FreeBSD ports
+      $plugin_prefix       = 'rubygem-smart_proxy_'
+      $foreman_api_package = 'rubygem-apipie-bindings'
```

Comment:
Yes, see http://portscout.freebsd.org/mmoll@freebsd.org.html, DragonFly uses the same ports tree, so everything is named like on FreeBSD.